modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -39,6 +39,107 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-176607. Verify that is not possible to edit an organization information without EVP Manager role"
+	@priority = 5
+	test CanNotEditWithoutManagerRole {
+		task ("Given I have created a request for a new Organization") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "employee@liferay.com",
+				userLoginFullName = "Employee User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+
+			Button.click(button = "Create an Organization");
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "12345-6",
+				city = "Organization City",
+				contactEmail = "contact@organization.com",
+				contactName = "Contact Name",
+				contactPhone = "12 345678901",
+				country = "Country Site",
+				organizationName = "Organization Test",
+				organizationSite = "organization.com",
+				smallDescription = "Small Description",
+				state = "Organization State",
+				street = "Organization Street",
+				taxId = 12345,
+				zipCode = "12345-678");
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("And the Organization is aprooved by Manager and Finance Users") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "Manager User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+
+			EVPOrganization.kebabMenuOrganization(organizationName = "Organization Test");
+
+			MenuItem.click(menuItem = "Review");
+
+			EVPOrganization.changeStatus(
+				button = "Approve",
+				typeMessage = "Your organization has been approved!");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "finance@liferay.com",
+				userLoginFullName = "Finance User");
+
+			EVPUtils.gotoEVP();
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+
+			EVPOrganization.kebabMenuOrganization(organizationName = "Organization Test");
+
+			MenuItem.click(menuItem = "Review");
+
+			EVPOrganization.changeStatus(
+				button = "Approve",
+				typeMessage = "Your organization has been approved!");
+
+			EVPNavigator.goToNavItem(navItem = "Organization");
+		}
+
+		task ("When I open the kebab menu at the Organization ellipsis button") {
+			EVPOrganization.viewStatus(
+				organizationName = "Organization Test",
+				status = "Verified");
+
+			EVPOrganization.kebabMenuOrganization(organizationName = "Organization Test");
+		}
+
+		task ("Then I can see that there isn't an Edit button") {
+			MenuItem.viewNotPresent(menuItem = "Edit");
+		}
+
+		task ("When I log in as Employee User") {
```

Comment:
@heiditrindade Hi, what u think about try to use, Given, When, Then, one time each?

Reply:
Hi, @styller1! I think in this test is not a case of doing another Given, When, Then. As I explain to you, in person, this test was supposed to be two tests before, but, since it was only a matter of doing a simple log out, log in and assert, as I talked to Nikki about it, we decided this should be a single test. Therefore, the double "then".